buildpack/lib/install_go.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Borrowing this script from the Node.js Buildpack.
+# https://raw.githubusercontent.com/cloudfoundry/nodejs-buildpack/master/scripts/install_go.sh
+
+set -euo pipefail
+
+GO_VERSION="1.14"
+
+if [ $CF_STACK == "cflinuxfs3" ]; then
```

Comment:
Add double quotes around `$CF_STACK`.

Reply:
I have done this 2x already ... the case of the Great Disappearing CF Stack Double Quotes

Reply:
HA I just lost them a 3rd time and I finally figured out why ... fixing